controllers/yawol-controller/loadbalancer/loadbalancer_controller.go

```diff
@@ -42,6 +42,7 @@ import (
 const (
 	DefaultRequeueTime = 10 * time.Millisecond
 	ServiceFinalizer   = "yawol.stackit.cloud/controller2"
+	HTTP_NOT_FOUND int	= 404
```

Comment:
Is there no const like this in some package already?

Reply:
Nope : D weather in our code, nor in the `gophercloud` package.

Reply:
i dropped the Const, in check inline for `404`, its only used here once rn.